### PR TITLE
Created Business Models

### PR DESCRIPTION
**Pull request by injacmarko**

### CODE REVIEW

**Review by FilipZecevic — APPROVED** [review]

File: src/BuisinessLogic/Models/CreateFuelLogDTO.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+namespace BusinessLogic
+{
+    public class CreateFuelLogDTO
+    {
+        public int Litres;
+        public int Cost;
+        public string TypeOfFuel;
```
Comment (djordjevics) [inline src/BuisinessLogic/Models/CreateFuelLogDTO.cs]: Enum, please

File: src/BuisinessLogic/Models/CreateFuelLogDTO.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+        public int TruckId;
```
Comment (djordjevics) [inline src/BuisinessLogic/Models/CreateFuelLogDTO.cs]: truck ?

What about car, motorcycle?

File: src/BuisinessLogic/Models/CreateFuelLogDTO.cs (hunk 1 above)
Comment (djordjevics) [reply src/BuisinessLogic/Models/CreateFuelLogDTO.cs]: type of fuel for fuel log?
One fuel log can be petrol, next fill can be diesel?

File: src/BuisinessLogic/Models/CreateVehicleDTO.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+namespace BusinessLogic
+{
+    public class CreateVehicleDTO
+    {
+        public string Model;
+        public string LicensePlate;
+        public string Color;
+        public int WheelCount;
+    }
+}
```
Comment (djordjevics) [inline src/BuisinessLogic/Models/CreateVehicleDTO.cs]: missing fuel type

File: src/BuisinessLogic/Models/FuelLogDTO.cs [hunk 4]
```diff
@@ -0,0 +1,11 @@
+namespace BusinessLogic
+{
+    public class FuelLogDTO
+    {
+        public int Id;
+        public int Litres;
+        public int Cost;
+        public string TypeOfFuel;
+        public int TruckId;
```
Comment (djordjevics) [inline src/BuisinessLogic/Models/FuelLogDTO.cs]: similar to CreateFuelLogDTO

File: src/BuisinessLogic/Models/FuelLogDTO.cs [hunk 5]
```diff
@@ -0,0 +1,11 @@
+namespace BusinessLogic
+{
+    public class FuelLogDTO
+    {
+        public int Id;
+        public int Litres;
+        public int Cost;
```
Comment (djordjevics) [inline src/BuisinessLogic/Models/FuelLogDTO.cs:12]: Currency

File: src/BuisinessLogic/Models/GetFuelLogByVehicleDTO.cs [hunk 6]
```diff
@@ -0,0 +1,7 @@
+namespace BusinessLogic
+{
+    public class GetFuelLogByVehicleDTO
```
Comment (djordjevics) [inline src/BuisinessLogic/Models/GetFuelLogByVehicleDTO.cs]: byVehicleId

File: src/BuisinessLogic/Models/VehicleDTO.cs [hunk 7]
```diff
@@ -0,0 +1,11 @@
+namespace BusinessLogic
+{
+    public class VehicleDTO
+    {
+        public int Id;
+        public string LicensePlate;
+        public string Model;
+        public string Color;
+        public int WheelCount;
```
Comment (djordjevics) [inline src/BuisinessLogic/Models/VehicleDTO.cs:12]: FuelType

**Review by djordjevics — CHANGES_REQUESTED** [review]

File: src/BuisinessLogic/Models/FuelLogDTO.cs [hunk 8]
```diff
@@ -0,0 +1,42 @@
+namespace BusinessLogic
+{
+    public class FuelLogDTO
+    {
+        public int Id;
+        public int Odometer;
+        public enum OdometerUnit
```
Comment (djordjevics) [inline src/BuisinessLogic/Models/FuelLogDTO.cs]: separate file

File: src/BuisinessLogic/Models/FuelLogDTO.cs [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+        {
+            km,
```
Comment (djordjevics) [inline src/BuisinessLogic/Models/FuelLogDTO.cs]: UPPERCASE

File: src/BuisinessLogic/Models/FuelLogDTO.cs [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+            mi
+        }
+        public int AmountFilled;
+        public enum AmountFilledUnit
```
Comment (djordjevics) [inline src/BuisinessLogic/Models/FuelLogDTO.cs]: separate file

File: src/BuisinessLogic/Models/Enums/CostCurrency.cs [hunk 11]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
```
Comment (djordjevics) [inline src/BuisinessLogic/Models/Enums/CostCurrency.cs]: unused usings

File: src/BuisinessLogic/Models/Enums/AmountFilledUnit.cs [hunk 12]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
```
Comment (djordjevics) [inline src/BuisinessLogic/Models/Enums/AmountFilledUnit.cs]: unused usings

File: src/BuisinessLogic/Models/Enums/FuelType.cs [hunk 13]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
```
Comment (djordjevics) [inline src/BuisinessLogic/Models/Enums/FuelType.cs]: unused usings

File: src/BuisinessLogic/Models/Enums/OdometerUnit.cs [hunk 14]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
```
Comment (djordjevics) [inline src/BuisinessLogic/Models/Enums/OdometerUnit.cs]: unused usings

**Review by djordjevics — CHANGES_REQUESTED** [review]

**Review by djordjevics — APPROVED** [review]

**Pull request merged**